libs/GasBurnerControl/GasBurnerControl.cpp

```diff
@@ -0,0 +1,291 @@
+#include "GasBurnerControl.h"
+#include "Arduino.h"
+
+
+GasBurnerControl::GasBurnerControl(byte powerPin, byte dejamPin, byte jammedPin, byte valvePin, byte ignitionPin) {
+    this->_powerPin = powerPin;
```

Comment:
Stupid question, why the `this` all the time? The instance dereferences them automatically, so no need to specify them.

Reply:
Removed in d9eb9b0